app/templates/admin.hbs

```diff
@@ -1,92 +1,115 @@
-<div class="admin-container">
-  <h2>Challenge Form</h2>
-  <form class="challenge-form" {{action "addChallenged" on="submit"}}>
-      <fieldset>
-          <h3>Add Challenge</h3>
-          <label>
-              Player 1:
-              {{select-component 
-              options=unchallengedPlayers 
-              selectOption="selectChallenger1"}}
-          </label>
-          <label>
-              Player 2:
-              {{select-component 
-              options=unchallengedPlayers 
-              selectOption="selectChallenger2"}}
-          </label>
-      </fieldset>
-      <fieldset>
-        <input type="submit" value="submit" class="submitButton">
-      </fieldset>
-  </form>
-</div>
-<div class="admin-container">
-  <h2>Add Game</h2>
+{{#if isAuthenticated}}
```

Comment:
not sure, might have some errors due to firebase not being defined as its new. i think the app is mvp ready